lib/order.rb

```diff
@@ -0,0 +1,37 @@
+class Order
+  attr_reader :summary, :menu
+
+  def initialize(menu)
+    @summary = Hash.new(0)
+    @menu = menu
```

Comment:
You could use dependency injection here to improve the user's experience by removing the need to create a menu object separately.

something like
```ruby
def initialize(menu = Menu.new)
  @summary = Hash.new(0)
  @menu = menu
end
```